mscore/workspace.cpp

```diff
@@ -1227,7 +1210,25 @@ void WorkspacesManager::initWorkspaces()
                   break;
                   }
             }
+      
       retranslate(m_workspaces);
+      
```

Comment:
I guess it's the moving of this code that changed the behavior of with respect to setting currentWorkspace and thus fixed the crash on reset of edited default workspace in at least some cases>